frontend/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
```

Comment:
are you using service workers?

Reply:
We are not using it now, but we will use the service workers when the app is little stable.